criu/uffd.c

```diff
@@ -1249,14 +1249,14 @@ static void lazy_pages_summary(struct lazy_pages_info *lpi)
 	lp_debug(lpi, "UFFD transferred pages: (%ld/%ld)\n",
 		 lpi->copied_pages, lpi->total_pages);
 
-#if 0
+
 	if ((lpi->copied_pages != lpi->total_pages) && (lpi->total_pages > 0)) {
```

Comment:
The same information is already printed above without the frightening text ;-)

The real problem if that `lpi->total_pages` only shows the initial count of the pages at the dump time, but it is not updated when some pages are unmapped or dropped with `madvise(MADV_DONTNEED)`.

Reply:
Thank you Mike, I have dropped this commit.

When running the lazy-thp test it fails but not always, and when it fails the output in `lazy-pages.log` is as follows:

successful: [lazy-pages.log](https://github.com/checkpoint-restore/criu/files/6058187/lazy-pages-log-1.txt)
```
UFFD transferred pages: (1033/1033)
```

failed: [lazy-pages.log](https://github.com/checkpoint-restore/criu/files/6058188/lazy-pages-log-2.txt)
```
UFFD transferred pages: (137/1033)
```